lib/contentful.js

```diff
@@ -142,46 +151,54 @@ module.exports.fetchCampaignIdsWithKeywords = function () {
 module.exports.fetchKeyword = function (keyword) {
   logger.debug(`contentful.fetchKeyword:${keyword}`);
 
-  return exports.fetchSingleEntry({
-    content_type: 'keyword',
-    'fields.keyword': keyword,
-  });
+  const query = exports.getQueryBuilder()
+    .type('keyword')
+    .keyword(keyword)
+    .build();
+  return exports.fetchSingleEntry(query);
+};
+
+module.exports.fetchKeywords = function fetchKeywords() {
+  logger.debug('contentful.fetchKeywords');
+  const query = exports.getQueryBuilder()
+    .type('keyword')
+    // We store keywords for both Thor and Production as entries, so filter by current environment.
+    .env(process.env.NODE_ENV)
+    .build();
+  return exports.getClient().getEntries(query)
+  .then(response => response.items.map(item => exports.formatKeywordFromResponse(item)))
+  .catch(error => exports.contentfulError(error));
 };
 
 module.exports.fetchKeywordsForCampaignId = function (campaignId) {
   logger.debug(`contentful.fetchKeywordsForCampaignId:${campaignId}`);
 
-  // TODO: refactor into a function that creates the query.
-  const query = {
-    content_type: 'keyword',
+  const query = exports.getQueryBuilder()
+    .type('keyword')
     // We store keywords for both Thor and Production as entries, so filter by current environment.
-    'fields.environment': process.env.NODE_ENV,
-  };
-  if (campaignId) {
-    query['fields.campaign.sys.contentType.sys.id'] = 'campaign';
-    query['fields.campaign.fields.campaignId'] = campaignId;
-  }
-
+    .env(process.env.NODE_ENV)
+    .custom({
+      'fields.campaign.sys.contentType.sys.id': 'campaign',
+      'fields.campaign.fields.campaignId': campaignId,
```

Comment:
This is where I'm a little bit on the fence on the `QueryBuilder` class abstraction. I can see how it's useful for the simple fetch queries like `fetchKeyword` or `fetchCampaign` to not need to know the inner workings of the Contentful content types, but the super consistent maybe OCD brain would prefer the Builder functions to match the field names, e.g. `campaignId` or `broadcastId` instead of `campaign`, `broadcast`, `environment` instead of `env`, `contentType` instead of `type`... 

Yet one still needs to know the inner workings of the Contentful content types to execute a complex query anyway with `custom`.  The devil's advocate question is does the `Builder` actually make things eaiser?

cc @sergii-tkachenko